docs/run-preinstalled-wda.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Run Preinstalled WebDriverAgentRunner
+---
+
+XCUITest driver can launch preinstalled WebDriverAgent directly against a real device.
+It lets you start an XCUITest driver session without the `xcodebuild` command execution to improve the session start performance.
+
+## For Real Devices
+
+### Capabilities
+
+- Required
+  - `appium:usePreinstalledWDA` to start a WebDriverAgent Runner-process without xcodebuild
+- Optional
+  - `appium:updatedWDABundleId` to customize the WebDriverAgent bundle id XCUITest driver launches
+
+### Example steps with Xcode
+
+1. Run WebDriverAgent with Xcode as test
```

Comment:
This probably needs some more detailed description, like:

- Open WebDriverAgent in Xcode
- Select WebDriverAgentRunner project
- Choose the target device
- Select Product->Test from the main menu bar

Reply:
Added to guide to With Xcode below and it has more concrete steps